avocado/utils/podman.py

```diff
@@ -49,11 +49,15 @@ async def execute(self, *args):
         :rtype: tuple with returncode, stdout and stderr.
         """
         try:
+            LOG.debug("Executing %s", args[0])
             proc = await create_subprocess_exec(self.podman_bin,
                                                 *args,
                                                 stdout=subprocess.PIPE,
                                                 stderr=subprocess.PIPE)
             stdout, stderr = await proc.communicate()
+            LOG.debug("Return code: %s", proc.returncode)
+            LOG.debug("Stdout: %s", stdout.decode())
```

Comment:
Depending on what the process generates on its stdout (or stderr) the decode may fail because of `errors=strict`.  Because this is for humans and informational purposes only, let's use "replace" to avoid raising `UnicodeDecodeError`.

Reply:
Fixed with the force-push.